src/twitch.py

```diff
@@ -142,14 +145,17 @@ async def bgdailii(ctx):
 async def goodBot(ctx):
     await ctx.send('MrDestructoid Just doing my job MrDestructoid')
 
-@twitchBot.command(name='wall_lii')
-async def wall_lii(ctx):
-    await ctx.send('HeyGuys I\'m a bot that checks the BG leaderboard to get data about player ranks and daily MMR fluctuations. I reset daily at Midnight CA time. Try using !bgrank [name] and !bgdaily [name] and !yesterday [name]. Also try !buddy [hero] and !goldenbuddy [hero] for buddy info')
-
-@twitchBot.command(name='help')
+@twitchBot.command(name='help', aliases=['wall_lii'])
 async def help(ctx):
     await ctx.send('HeyGuys I\'m a bot that checks the BG leaderboard to get data about player ranks and daily MMR fluctuations. I reset daily at Midnight CA time. Try using !bgrank [name] and !bgdaily [name] and !yesterday [name]. Also try !buddy [hero] and !goldenbuddy [hero] for buddy info')
 
+@twitchBot.event()
+async def event_error(error, data):
+    if isinstance(error, CommandNotFound):
+        return
```

Comment:
Should we print something helpful here? At least for the time being to make sure this solution is working at suppressing these execptions?

Reply:
isn't the goal to suppress these exceptions. CommandNotFound just means it's probably a command for another bot